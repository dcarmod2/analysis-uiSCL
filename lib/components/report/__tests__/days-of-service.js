//
import enzyme from 'enzyme'
import React from 'react'

import DaysOfService from '../days-of-service'

describe('Report > DaysOfService', () => {
  it('renders correctly', () => {
    const props = {
      monday: true,
      tuesday: true
    }

    const tree = enzyme.shallow(<DaysOfService {...props} />)
    expect(tree).toMatchSnapshot()
  })
})
