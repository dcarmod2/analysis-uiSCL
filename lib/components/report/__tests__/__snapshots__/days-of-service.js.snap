// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report > DaysOfService renders correctly 1`] = `
<ForwardRef
  isInline={true}
>
  <Text
    color="black"
    key="monday"
    textDecoration="none"
  >
    Mo
  </Text>
  <Text
    color="black"
    key="tuesday"
    textDecoration="none"
  >
    Tu
  </Text>
  <Text
    color="gray"
    key="wednesday"
    textDecoration="line-through"
  >
    We
  </Text>
  <Text
    color="gray"
    key="thursday"
    textDecoration="line-through"
  >
    Th
  </Text>
  <Text
    color="gray"
    key="friday"
    textDecoration="line-through"
  >
    Fr
  </Text>
  <Text
    color="gray"
    key="saturday"
    textDecoration="line-through"
  >
    Sa
  </Text>
  <Text
    color="gray"
    key="sunday"
    textDecoration="line-through"
  >
    Su
  </Text>
</ForwardRef>
`;
